pkg/cloudprovider/gce/gce.go

```diff
@@ -500,6 +507,11 @@ func (gce *GCECloud) ExternalID(instance string) (string, error) {
 	return strconv.FormatUint(inst.Id, 10), nil
 }
 
+// InstanceID returns the cloud provider ID of the specified instance.
+func (gce *GCECloud) InstanceID(instance string) (string, error) {
+	return gce.projectID + "/" + gce.zone + "/" + instance, nil
```

Comment:
Just call canonicalizeInstanceName(instance) for now. Maybe we can verify via metadata server later.